packages/zambdas/src/patient/appointment/create-appointment/index.ts

```diff
@@ -95,21 +97,31 @@ export const index = wrapHandler(async (input: ZambdaInput): Promise<APIGatewayP
     console.groupEnd();
     console.debug('validateRequestParameters success', JSON.stringify(validatedParameters));
 
-    if (!zapehrToken) {
+    if (!oystehrToken) {
       console.log('getting token');
-      zapehrToken = await getAuth0Token(input.secrets);
+      oystehrToken = await getAuth0Token(input.secrets);
     } else {
       console.log('already have token');
     }
-    const oystehr = createOystehrClient(zapehrToken, input.secrets);
+    const oystehr = createOystehrClient(oystehrToken, input.secrets);
 
     console.time('performing-complex-validation');
     const effectInput = await createAppointmentComplexValidation(validatedParameters, oystehr);
-    const { slot, scheduleOwner, serviceMode, patient, questionnaireCanonical, visitType } = effectInput;
+    const {
+      slot,
+      scheduleOwner,
+      serviceMode,
+      patient,
+      questionnaireCanonical,
+      visitType,
+      appointmentMetadata: maybeMetadata,
+    } = effectInput;
     console.log('effectInput', effectInput);
     console.timeEnd('performing-complex-validation');
 
-    console.log('creating appointment');
+    console.log('creating appointment with metadata: ', JSON.stringify(maybeMetadata, null, 2));
```

Comment:
```suggestion
    console.log('creating appointment with metadata: ', JSON.stringify(maybeMetadata));
```
it's easier to follow in aws/oystehr consoles when they're in one line

Reply:
i find it easier for me to copy one line and make my ide format it correctly than to scroll endlessly when the log stops being relevant to me